src/main/java/at/ac/tuwien/kr/alpha/grounder/NaiveGrounder.java

```diff
@@ -484,7 +483,8 @@ public int register(NoGood noGood) {
 				throw oops("Grounded atom should be ground but is not");
 			}
 
-			if (factsFromProgram.get(substitutedAtom.getPredicate()) == null || !factsFromProgram.get(substitutedAtom.getPredicate()).contains(new Instance(substitutedAtom.getTerms()))) {
+			if (factsFromProgram.get(substitutedAtom.getPredicate()) == null
```

Comment:
The many formatting changes in this file (and other files) make it hard to see whether anything besides formatting has changed at all. Please format only files (or, even better, lines) where you actually change semantics.

Reply:
Sorry for all the formatting fails! I adapted my formatted config a few weeks ago, new commits should be better. For this file, I just simply reverted to the current version from master, this should take care of it.